src/web/backend/nextcloud/api.py

```diff
@@ -36,32 +54,49 @@ class UserViewSet(NextCloudMixin,
     def get(self, username=None):
         """ List users """
         if username is None:
-            res = self.nextcloud.get_users()
-            return self.nxc_response(res)
+            res = self.edap.get_users()
```

Comment:
I would appreciate an interface between TEAP and EDAP. In other words, what do you thing of introducing a wrapper that would convert the return value of `get_users` to a list of some `User` class instances?
You do this to some degree in the `edap_to_dict` function, but I think that you can go a bit further.

Reply:
Agree, will add serializer for this.

Reply:
Great idea, will add some kind of serializers in separate PR.